clang/lib/Interpreter/Interpreter.cpp

```diff
@@ -597,6 +597,9 @@ llvm::Error Interpreter::CreateExecutor() {
     auto JTMB = createJITTargetMachineBuilder(TT);
     if (!JTMB)
       return JTMB.takeError();
+#if defined(_WIN32)
```

Comment:
I'm not sure what the common practice is for the JIT/interpreter side, but I would prefer to use a check in code, whether the target triple is windows, rather than using an ifdef. Most of clang is cross-compiling agnostic anyway. For the JIT that's obviously not the case, but by not using ifdefs, you get the code compile tested in all builds, not only the windows ones.

Reply:
In fact the JIT supports cross-compiling too, and will happily run in cross-process mode targeting a process on different OS / architecture. :)